src/constants/wallets.ts

```diff
@@ -17,13 +18,17 @@ export const SUPPORTED_WALLETS: { [key: string]: WalletInfo } = {
     description: 'Connect to Trust Wallet, Rainbow Wallet and more...',
     href: null,
   },
-  COINBASE_WALLET: {
-    connector: coinbaseWallet,
-    name: 'Coinbase Wallet',
-    iconType: 'coinbaseWallet',
-    description: 'Use Coinbase Wallet app on mobile device',
-    href: null,
-  },
+  // Hide coinbase wallet option in mobile device
+  // Ticket EB-3000131432
+  ...(!isMobileDevices() && {
+    COINBASE_WALLET: {
+      connector: coinbaseWallet,
+      name: 'Coinbase Wallet',
+      iconType: 'coinbaseWallet',
+      description: 'Use Coinbase Wallet app on mobile device',
```

Comment:
Since we are hiding the wallet when mobile, we should show a different description here.

Reply:
Code updated